src/js/content_scripts/input-mapper-utils.js

```diff
@@ -188,4 +188,156 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
             }, that.options.frequency);
         }
     };
+
+    /**
+     *
+     * Tab through the webpage using analog sticks.
+     *
+     * @param {Object} that - The inputMapper component.
+     * @param {Integer} value - The value of the gamepad input.
+     * @param {Integer} speedFactor - Times by which the tabbing speed should be increased.
+     * @param {Boolean} invert - Determine if the tabbing should be in opposite order.
+     *
+     */
+    gamepad.inputMapperUtils.analogTabbing = function (that, value, speedFactor, invert) {
+        var inversionFactor = invert ? -1 : 1;
+        value = value * inversionFactor;
+        if (value > 0.20) {
+            clearInterval(that.options.members.intervalRecords.reverseTab);
+            that.forwardTab(value, speedFactor);
+        }
+        else if (value < -0.20) {
+            clearInterval(that.options.members.intervalRecords.forwardTab);
+            that.reverseTab(-1 * value, speedFactor);
+        }
+        else {
+            clearInterval(that.options.members.intervalRecords.forwardTab);
+            clearInterval(that.options.members.intervalRecords.reverseTab);
+        }
+    };
+
+    /**
+     *
+     * Focus on the next tabbable element.
+     *
+     * @param {Object} that - The inputMapper component.
+     * @param {Integer} value - The value of the gamepad input.
+     * @param {Integer} speedFactor - Times by which the tabbing speed should be increased.
+     *
+     */
+    gamepad.inputMapperUtils.forwardTab = function (that, value, speedFactor) {
+        // Stop tabbing for the previous input value.
+        clearInterval(that.options.members.intervalRecords.forwardTab);
+
+        if (value > that.options.cutoffValue) {
```

Comment:
I'm not sure why you have the bounds checking twice for analog controls.  Seems like you should only have it once, and in the analogue function.